javascript/nanoexpress-pro/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "nanoexpress-pro": "^1.2.8"
```

Comment:
could you use `~1.2.8` to keep in `1.2` ?

Reply:
But i want upgrade to latest version as possible when i change happens in versions

Reply:
I understand, but to display the version in results, I need a version range (in a **minor**).

There is a @dependabot that create `PR` for updates

Reply:
Ok, i fix it :)